src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+import { FilterTodos } from '../../enum/FilterTodos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  setChooseFilter: (value: string) => void,
+  todos: Todo[],
+  chooseFilter: string,
+  deletePerformedTask: () => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  setChooseFilter,
+  todos,
+  chooseFilter,
+  deletePerformedTask,
+}) => {
+  const activeButton = () => {
+    return todos.find(todo => todo.completed);
```

Comment:
the same here, better to use `some` method, 'cause you don't need the todo information and fields, you only want to know if this todo exists in the array